src/components/ImageCarousel.tsx

```diff
@@ -55,7 +55,7 @@ const ImageCarousel = ({ images }: ImageCarouselPropsType): ReactElement => {
   }
 
   return (
-    <ImageView>
+    <ImageView testID={'Swipeable'}>
```

Comment:
We already have the testID "Swipeable" used in HomeScreen, not sure if it matters if duplicated.

Reply:
should not lead to problems as those components don't share a screen it is possible to have multiple testIds with the same name.